testing_and_setup/compass/ocean/hurricane/forward_template.xml

```diff
@@ -30,6 +30,8 @@
                 <option name="config_tidal_potential_reference_time">'2012-10-10_00:00:00'</option>
                 <option name="config_tidal_potential_ramp">10.0</option>
 
```

Comment:
I don't think we need this white space here in this file.  Can you please remove?

Reply:
I was trying to keep different namelist groups separated because I think it's more readable. However, if you disagree, I'll change it.

Reply:
You are right, best to leave as-is.